kie-spring/src/test/java/org/kie/spring/CodeVersion.java

```diff
@@ -18,6 +18,6 @@
 public class CodeVersion {
 
     // Do not modify this: this is automatically incremented by the maven replacer plugin
-    public final static String VERSION = "7.0.0-SNAPSHOT";
+    public final static String VERSION = "7.1.0-SNAPSHOT";
```

Comment:
@markcoble I'd opt are removing this file from this PR as it's not related to the changes.

@psiroky frankly speaking I have not idea what this class is suppose to do... anyway it's a test class so not big impact I believe